src/register-org/components/check-your-answers/check-your-answers.component.html

```diff
@@ -0,0 +1,194 @@
+<main id="content" class="govuk-main-wrapper govuk-!-margin-top-4" role="main" tabindex="-1">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <a class="govuk-back-link" href="javascript:history.back()">Back</a>
+      <h1 class="govuk-heading-l">Check your answers before you register</h1>
+      <h2 class="govuk-heading-m">Organisation details</h2>
+
+      <dl class="govuk-summary-list govuk-!-margin-bottom-0">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Organisation name</dt>
+          <dd class="govuk-summary-list__value">{{ registrationData.name }}</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="javascript:void(0)"
+              [routerLink]="'/register-org-new/company-house-details'">Change
+              <span class="govuk-visually-hidden">Organisation name</span>
+            </a>
+          </dd>
+        </div>
+      </dl>
+
+      <dl *ngIf="registrationData.companyHouseNumber" class="govuk-summary-list govuk-!-margin-bottom-0">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Company registration number</dt>
+          <dd class="govuk-summary-list__value">{{ registrationData.companyHouseNumber }}</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="javascript:history.back()"
+              [routerLink]="'/register-org-new/company-house-details'">
+              Change<span class="govuk-visually-hidden">Company registration number</span>
+            </a>
+          </dd>
+        </div>
+      </dl>
+
+      <dl class="govuk-summary-list govuk-!-margin-bottom-0"
+        [ngClass]="{ 'govuk-summary-list--no-border':registrationData.dxNumber}">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Organisation address</dt>
+          <dd class="govuk-summary-list__value">{{ registrationData.address }}</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="javascript:void(0)">
+              Change<span class="govuk-visually-hidden">Organisation address</span>
+            </a>
+          </dd>
+        </div>
+      </dl>
+
+      <dl *ngIf="registrationData.dxNumber" class="govuk-summary-list govuk-!-margin-bottom-0">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">DX reference</dt>
+          <dd class="govuk-summary-list__value">{{ registrationData.dxNumber }}</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="javascript:void(0)"
+              [routerLink]="'/register-org-new/document-exchange-reference-details'">
```

Comment:
@DavidJayakumar - please use this property `REGISTER_ORG_NEW_ROUTE` from `RegisterOrgService` as suggested by Ronald for one of my older PR.